extensions.json

```
@@ -1003,8 +1003,8 @@
     },
     {
       "id": "marus25.cortex-debug",
-      "download": "https://github.com/Marus/cortex-debug/releases/download/V0.4.4/cortex-debug-0.4.4.vsix",
-      "version": "0.4.4"
+      "download": "https://github.com/Marus/cortex-debug/releases/download/V0.4.5-pre1/cortex-debug-0.4.5-pre1.vsix",
+      "version": "0.4.5-pre1"
```

Let's wait for the next (not-pre-) release

```suggestion
      "download": "https://github.com/Marus/cortex-debug/releases/download/V0.4.4/cortex-debug-0.4.4.vsix",
      "version": "0.4.4"
```